src/analysis/output.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+/**
+ * This module defines a rich output format for cred scores, so that
+ * we can use it to drive UIs and data analysis.
+ */
+import type {Alias} from "../plugins/identity/alias";
+import type {EdgeWeight} from "../core/weights";
+
+// unenforced
+export type Integer = number;
+export type TimestampMs = Integer;
+export type Index = Integer;
+
+/**
+ * Describes an individual node in the contribution graph.
+ * Includes the information in the raw Graph node, along
+ * with scoring information, i.e. its total cred, and
+ * the amount of cred that it minted.
+ *
+ * While we are in TimelineCred, the "cred" field will have the total cred,
+ * summed across all time slices. Once we migrate to CredRank, it will just be
+ * the cred directly calculated by the CredRank algorithm.
+ */
+export type OutputNode = {|
+  +address: $ReadOnlyArray<string>,
+  +cred: number,
+  +mint: number,
```

Comment:
Some preference to call it `mints`.
Since this is presented as a current state snapshot, I think a present tense is fine.

Reply:
Reasonable.